packages/esm-service-queues-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -126,13 +127,13 @@ function OldQueueTable({ queueEntries }: { queueEntries: QueueEntry[] }) {
   const { t } = useTranslation();
   const currentServiceName = useSelectedServiceName();
   const currentQueueLocation = useSelectedQueueLocationUuid();
-  const { queues } = useQueues(currentQueueLocation);
   const { visitQueueNumberAttributeUuid } = useConfig<ConfigObject>();
+  const currentServiceUuid = useSelectedServiceUuid();
   const visitQueueEntries = queueEntries.map((entry) =>
     mapVisitQueueEntryProperties(entry, visitQueueNumberAttributeUuid),
   );
+  const queues = uniqBy(queueEntries?.flatMap((entry) => entry.queue) ?? [], 'uuid');
```

Comment:
I think this will make the dropdown list display only queues that have at least one queue entry. Are we sure we want that? The order of those queues can be unpredictable based on the order of the returned queue entries.